ci/docker/runtime_functions.sh

```diff
@@ -804,6 +804,14 @@ unittest_ubuntu_cpu_clojure() {
     ./contrib/clojure-package/ci-test.sh
 }
 
+unittest_ubuntu_cpu_clojure_nightly() {
```

Comment:
Please rename to integration tests instead of nightly to reflect the workload of the task :)